core/src/settings.ts

```diff
@@ -245,3 +245,8 @@ export const NONCE_MANAGER_JOB_SETTINGS = {
 export function getObservedBlockRedisKey(contractAddress: string) {
   return `${contractAddress}-listener-${DEPLOYMENT_NAME}`
 }
+
+export const NONCE_MANAGER_POLLING_INTERVAL = 1_000 // ms
+export const NONCE_MANAGER_SLACK_FREQUENCY_TIME = 10 // min
+export const NONCE_MANAGER_SLACK_FREQUENCY_RETRIES =
+  (NONCE_MANAGER_SLACK_FREQUENCY_TIME * 60) / (NONCE_MANAGER_POLLING_INTERVAL / 1000)
```

Comment:
@Intizar-T If you want to assign unit to a constant it is better to include it to constant name, otherwise, other developers can learn about it by coming to this file.

```suggestion
export const NONCE_MANAGER_POLLING_INTERVAL_MS = 1_000
export const NONCE_MANAGER_SLACK_FREQUENCY_TIME_MIN = 10
export const NONCE_MANAGER_SLACK_FREQUENCY_RETRIES =
  (NONCE_MANAGER_SLACK_FREQUENCY_TIME_MIN * 60) / (NONCE_MANAGER_POLLING_INTERVAL_MS / 1000)
```